fedimg/consumers.py

```diff
@@ -26,13 +26,13 @@
 
 import fedmsg.consumers
 import fedmsg.encoding
-import koji
+import fedfind.release
```

Comment:
Looks like it is packaged for EPEL, yes:  https://apps.fedoraproject.org/packages/fedfind